IETF-RFC.md

```diff
@@ -394,6 +394,10 @@ If `multi` is given, one or more protocol
                 only support `webdav`.
 
    * Protocol details for `webdav` MAY contain:
+     * REQUIRED uri (string)
```

Comment:
why is this one moving from optional to required, was that discussed?

Reply:
In fact it has always been required (in spec.yaml), the IETF text was actually incorrect